gateway/app/static/styles/index.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Is this really necessary? Maybe we can make this change in a different PR

Reply:
Done! (5347e6e)